test/websocket/stream.cpp

```diff
@@ -8,11 +8,13 @@
 // Test that header file is self-contained.
 #include <beast/websocket/stream.hpp>
 
+#include "../fail_stream.hpp"
```

Comment:
Files that are part of unit tests are not reachable through any of the include directories listed in the compile switches (by design).